bundles/af-core/src/test/resources/form/formcontainer/test-forms-in-sites.json

```diff
@@ -0,0 +1,35 @@
+{
+  "jcr:primaryType": "cq:Page",
+  "jcr:content": {
+    "jcr:primaryType": "cq:PageContent",
+    "root": {
+      "jcr:primaryType": "nt:unstructured",
+      "sitecontainer": {
+        "jcr:primaryType": "nt:unstructured",
+        "formcontainer": {
+          "jcr:primaryType": "nt:unstructured",
+          "sling:resourceType": "core/fd/components/form/container/v2/container",
+          "thankyouPage": "/a/b/c",
+          "thankYouMessage": "Thank you for submitting.",
+          "thankYouOption": "message",
+          "redirect": "/content/wknd",
+          "prefillService": "FDM",
+          "fieldType": "form",
+          "schemaType": "jsonschema",
+          "prefillService" : "dummy",
+          "textinput": {
+            "jcr:primaryType": "nt:unstructured",
+            "sling:resourceType": "core/fd/components/form/textinput/v1/textinput",
+            "name": "abc",
+            "jcr:title": "def",
+            "hideTitle": false,
+            "description": "dummy",
+            "fieldType": "text-input",
+            "fd:translationIds": "{\"jcr:title\":\"guideContainer##textinput##jcr:title##5598\",\"placeholder\":\"guideContainer##textinput##description##5648\"}",
```

Comment:
You won't have fd:translationIds when a form is used inside sites ? Why do you need this

Reply:
yes, used an existing json hence it is appearing, not using this in the tests for sites.

Reply:
removed